### PR TITLE
Minor suggestions

### PR DESCRIPTION
**Pull request by lordrex34**
Well, if you like it kindly approve. Otherwise, please let me know your concerns.

Thanks.

### CODE REVIEW

File: Plugins-DB/src/main/java/com/github/unafraid/plugins/DBPluginRepository.java [hunk 1]
```diff
@@ -44,24 +44,32 @@
...
 	@Override
 	public void startAll()
 	{
 		getInstalledPlugins().forEach(plugin ->
 		{
-			try
+			if (plugin.setState(PluginState.INITIALIZED, PluginState.INSTALLED))
 			{
-				if (plugin.setState(PluginState.INITIALIZED, PluginState.INSTALLED))
+				try (PluginsDAO pluginsDao = DatabaseProvider.DBI.open(PluginsDAO.class))
```
Comment (UnAfraid) [inline Plugins-DB/src/main/java/com/github/unafraid/plugins/DBPluginRepository.java]: try catch outside the loop

File: Plugins-DB/src/main/java/com/github/unafraid/plugins/DBPluginRepository.java [hunk 2]
```diff
@@ -162,4 +170,19 @@ public void uninstallPlugin(AbstractDBPlugin plugin) throws PluginException
...
+	/**
+	 * Update plugin's auto start state.
+	 * @param plugin the plugin itself
+	 * @param autoStart if {@code true}, the plugin will start on application boot, if {@code false} the plugin won't
+	 */
+	public void updateAutoStart(AbstractDBPlugin plugin, boolean autoStart)
+	{
+		Objects.requireNonNull(plugin);
+		
+		try (PluginsDAO pluginsDao = DatabaseProvider.DBI.open(PluginsDAO.class))
```
Comment (UnAfraid) [inline Plugins-DB/src/main/java/com/github/unafraid/plugins/DBPluginRepository.java]: Don't do that, make a db check if plugin is present there, u won't even know if u updated or not, check if present if not throw exception if does then proceed

File: Plugins-DB/src/main/java/com/github/unafraid/plugins/db/dao/dto/Plugin.java [hunk 3]
```diff
@@ -82,4 +90,48 @@ public void setVersion(int version)
...
+	 */
+	public long getInstalledOn()
+	{
+		return installedOn;
+	}
+	
+	/**
+	 * @param installedOn the installedOn to set
+	 */
+	public void setInstalledOn(long installedOn)
```
Comment (UnAfraid) [inline Plugins-DB/src/main/java/com/github/unafraid/plugins/db/dao/dto/Plugin.java]: I don't think we gonna modify installed on date.

Comment (lordrex34) [inline Plugins-DB/src/main/java/com/github/unafraid/plugins/db/dao/dto/Plugin.java]: That's why I've asked - do we need the setters on other fields also or not?

File: Plugins-DB/src/main/java/com/github/unafraid/plugins/DBPluginRepository.java [hunk 4]
```diff
@@ -71,15 +85,15 @@ public void startAll()
...
+		getInstalledPlugins().filter(plugin -> plugin.getState() == PluginState.STARTED).forEach(plugin ->
 		{
 			try
 			{
 				plugin.stop();
 			}
 			catch (PluginException e)
 			{
-				LOGGER.warn("Failed to stop plugin {}", plugin.getName(), e);
+				LOGGER.warn("Failed to stop plugin " + plugin.getName(), e);
```
Comment (UnAfraid) [inline Plugins-DB/src/main/java/com/github/unafraid/plugins/DBPluginRepository.java]: ?

**Pull request merged**